internal/command/e2etest/automation_test.go

```diff
@@ -16,7 +16,7 @@ import (
 
 // The tests in this file run through different scenarios recommended in our
 // "Running Terraform in Automation" guide:
-//     https://www.placeholderplaceholderplaceholder.io/guides/running-terraform-in-automation.html
+//     https://developer.hashicorp.com/terraform/tutorials/automation/automate-terraform
```

Comment:
we currently do not have such a tutorial, we can keep the placeholder for now if we would like to